### PR TITLE
Heading change in /purge-caches

### PR DESCRIPTION
**Pull request by Soundharyaam**
**Issue:**

The heading was displayed as the `Cache Purges` in the `/purge-caches` route which mismatches the current navbar heading for this route.

**What did I do to resolve this:**

Hence, I made the heading for this route as `Purge Cache` to match the navbar.

### CODE REVIEW

**Comment by vishakhanihore** [comment]
There are multiple unwanted changes in the file which you just made to increase the line count. Also there is no issue created for the following PR. I think you didn't read the `Contributions Guidelines`.

File: ui/src/views/CachePurges/ViewCachePurges/index.jsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default class ViewCachePurges extends Component {
             </Typography>
           </HelpView>
         }
-        title="Cache Purges">
+        title="Purge Cache">
```
Comment (helfi92) [inline ui/src/views/CachePurges/ViewCachePurges/index.jsx]: Let's pluralize it: `Purge Caches`

File: ui/src/views/Tasks/NoTask/index.jsx [hunk 2]
```diff
@@ -48,7 +48,12 @@ export default class NoTask extends Component {
       <Dashboard
         title="View Tasks"
         helpView={<HelpView description={description} />}
-        search={<Search onSubmit={this.handleTaskSearchSubmit} />}>
+        search={
+          <Search
+            placeholder="Search Task ID"
+            onSubmit={this.handleTaskSearchSubmit}
+          />
+        }>
```
Comment (helfi92) [inline ui/src/views/Tasks/NoTask/index.jsx]: A rebase is needed.

**Comment by Soundharyaam** [comment]
Closing this PR, cause of the issue with rebasing.

Created the new one over here:
https://github.com/taskcluster/taskcluster/pull/1566

**Pull request closed without merging**